.travis.yml

```
@@ -1,19 +1,23 @@
 language: python
 python:
   - "3.6"
+  - "3.7"
 
 branches:
   only:
   - master
   - develop
-  - /^release-.*$/
+  - /^release[/-].*$/
+  - /^feature[/-].*$/
 
 install:
   - go get github.com/icon-project/rewardcalculator/cmd/icon_rc
-  - pip install -r requirements.txt
+  - virtualenv venv
+  - source venv/bin/activate
+  - pip install -e .[test]
 
 script:
-  - python setup.py test
+  - python -m pytest -rsxX
```

```suggestion
  - python -m pytest -r sxX
```

  -r chars        show extra test summary info as specified by chars: (f)ailed,
                        (E)rror, (s)kipped, (x)failed, (X)passed, (p)assed, (P)assed
                        with output, (a)ll except passed (p/P), or (A)ll. (w)arnings are
                        enabled by default (see --disable-warnings).

---

do you recommend use space after option name?
https://docs.pytest.org/en/latest/usage.html#detailed-summary-report
official documents show usage without "space".

---

good